tests/src/coffee_integration_tests.rs

```diff
@@ -32,6 +33,28 @@ pub async fn init_coffee_test() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+pub async fn init_coffee_test_cmd() -> anyhow::Result<()> {
+    init();
+
+    let args = MockCoffeeArgs {
+        conf: None,
+        data_dir: None,
+        network: Some("testnet".to_string()),
```

Comment:
```suggestion
        network: Some("regtest".to_string()),
```

Try this kind of network